app/parse.py

```diff
@@ -8,5 +13,47 @@ class Course:
     duration: str
 
 
+def parse_single_course(course: BeautifulSoup) -> Course:
+    return Course(
+        name=course.select_one(
+            "h1[data-qa='profession-title']"
+        ).text.split(":")[0],
+        short_description=course.select_one(
+            "p.typography_landingTextMain__Rc8BD."
+            "SalarySection_aboutProfession__1VFHK"
+        ).text,
+        duration=course.select_one(
+            ".ComparisonTable_tableBody__W5hzV.mb-24 "
+            "> div:nth-of-type(7) "
+            "> div:nth-of-type(2)"
+        ).text
+    )
+
+
+def get_courses_urls() -> list[str]:
+    page = requests.get(BASE_URL).content
```

Comment:
Consider adding error handling for the HTTP request to manage cases where the request fails or the response is not as expected.